resources/wiki-overrides.properties

```diff
@@ -80,6 +80,3 @@ handlebars=https://github.com/jenkinsci/js-libs/blob/master/handlebars/README.md
 jquery-detached=https://github.com/jenkinsci/js-libs/blob/master/jquery-detached/README.md
 momentjs=https://github.com/jenkinsci/js-libs/blob/master/momentjs/README.md
 numeraljs=https://github.com/jenkinsci/js-libs/blob/master/numeraljs/README.md
-
-# Required until a newer version than 1.0.0 is released
-synopsys-polaris=https://github.com/jenkinsci/synopsys-polaris-plugin
```

Comment:
@daniel-beck I missed this one; 1.1.0 is released with this change https://github.com/jenkinsci/synopsys-polaris-plugin/blob/1.1.0/build.gradle#L28